language/Portuguese_Brazil/strings.po

```diff
@@ -464,7 +464,7 @@ msgstr "Velocidade"
 
 #: src/frontend/mame/ui/submenu.cpp:60
 msgid "Adjust speed to match refresh rate"
-msgstr "Ajuste a velocidade para bater com a taxa de atualização"
+msgstr "Ajustar a velocidade pra combinar com a taxa da atualização"
```

Comment:
Usage of "pra" is too informal. You made this kind of change many times. Please keep it as "para", instead.

Also, it is better to say "taxa de atualização".